main.go

```diff
@@ -887,7 +887,7 @@ func (s *Server) injectRoute(path *bgptable.Path) error {
 // linux kernel
 // TODO: multipath support
 func (s *Server) watchBGPPath() error {
-	watcher := s.bgpServer.Watch(bgpserver.WatchBestPath())
+	watcher := s.bgpServer.Watch(bgpserver.WatchBestPath(false))
```

Comment:
@gunjan5 : no sure I follow?  there is no zero-value here since it's a fn argument which you have to specify.